src/app/shared/components/inline-value-edit/inline-value-edit.component.scss

```diff
@@ -120,6 +120,7 @@ select {
 .inline-value-textarea {
   min-height: 6 * 1.5rem;
   white-space: pre-wrap;
```

Comment:
```suggestion
```
I don't think we need the `white-space` attribute anymore since `overflow-wrap` (presumably?) overrides its behavior, and including both of them is a bit confusing.